documentation/docs/src/user-guide/wallet.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the different wallet concepts and options that are available to users of Namada who want to be able to [send, receive and interact](#send-and-receive-nam-tokens) with NAM tokens on the Namada blockchain.
 
+
+<!-- I want to either hyperlink or delete the below. I don't understand-->
+
```

Comment:
I would remove it as right now only the file wallet is used (afaik?) but would double check. Otherwise we have things in our documentation that are not in a release of namada (yet)